docs/developer.md

```diff
@@ -0,0 +1,17 @@
+# Developer Hub
+
+Welcome to the **Developer Hub** of Eclipse Tractus-X!
+
+This is the place, where you'll find information, specifically tailored for Tractus-X developers. It includes general
+information about implications of Tractus-X being an Eclipse Foundation project, tips on how to contribute and also
+rules for releasing software in Eclipse Tractus-X.
+
+In case you have questions about these rules just want to get in contact, feel free to check out the
+[Community](/community/intro) page. It sets the stage for your contribution to Eclipse Tractus-X and provides a broad
+overview of our project and community.
+
+Once you're familiar with our community, the **Developer Hub** becomes your critical next step. This space is your
```

Comment:
can we rephrase the "critical next step"? I know it was there in the old version, too. But it sounds a bit overly dramatic.

Reply:
Removed in [6188af9](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/583/commits/6188af9ac192b364ff48a2c0b3d7d16a7c5b1f0f)